packages/backend/src/core/tx-count/RpcBlockDownloader.ts

```diff
@@ -0,0 +1,67 @@
+import { Logger, TaskQueue } from '@l2beat/common'
+import { ProjectId, UnixTime } from '@l2beat/types'
+
+import { TxCountRepository } from '../../peripherals/database/TxCountRepository'
+import { EthereumClient } from '../../peripherals/ethereum/EthereumClient'
+import { Clock } from '../Clock'
+
+export class RpcBlockDownloader {
```

Comment:
This name is confusing. Primary focus of this tx is to count tx in block and insert into database

Reply:
BlockTxCountUpdater?